spec/fixtures/files/datahashes/core/collectionobject1.json

```diff
@@ -52,10 +49,10 @@
     "measuredPart": "image-size",
     "dimensionSummary": "charger ya samsung",
     "dimension": "circumference^^",
-    "measuredByPerson": "afa^^",
```

Comment:
With the delimiter on the end, we end up getting three rows worth of measuredBy values: 

- afa
- (blank)
- pandemic